aboutuspage.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>replit</title>
+  <link href="style.css" rel="stylesheet" type="text/css" />
+  <title> chirp website aboutus</title>
+</head>
+  
+
+<body>
+
+
+  <header>
+      <nav class="navbar">
+          <div class="logo">
+            <img src = "birdlogo.png" alt="bird logo">
+          </div>
+          <div class="menu">
+              <ul>
+                  <li><a href="#">AboutUs</a></li>
+                  <li><a href="#">Terms & Conditions</a></li>
```

Comment:
( li><a href="#">Terms &amp; Conditions</a></li ) 


use this instead the current code as that is triggering the linter to fail it.